json-viewer/jquery.json-viewer.js

```diff
@@ -68,8 +71,9 @@
         html += '[<ol class="json-array">';
         for (var i = 0; i < json.length; ++i) {
           html += '<li>';
-          // Add toggle button if item is collapsable
-          if (isCollapsable(json[i])) {
+          // Add toggle button if item is collapsable and not a big number
+          var ignoreItem = (options.bigNumbers && (typeof json[key].toExponential === 'function' || json[key].isLosslessNumber));
```

Comment:
Thanks for following up on your new feature!

I think we could avoid performing the big number check, and instead make a stricter type checking in `isCollapsable` :

```js
function isCollapsable(arg) {
  return (arg.constructor == Object && Object.keys(arg).length > 0) || (arg instanceof Array && arg.length > 0);
}
```

So it would return true only for raw JS objects (and arrays).